dev-qt/qtwebengine/qtwebengine-6.7.0.ebuild

```diff
@@ -70,6 +70,7 @@ RDEPEND="
 	designer? ( ~dev-qt/qttools-${PV}:6[designer] )
 	geolocation? ( ~dev-qt/qtpositioning-${PV}:6 )
 	kerberos? ( virtual/krb5 )
+	opengl? ( virtual/opengl )
```

Comment:
virtual/opengl is deprecated, the right dependency for GL/glx.h nowadays is `libglvnd[X]`. And it does not actually *need* libGL* at runtime, it's optfeature'ish and that's why libglvnd is currently DEPEND-only.

Also, did you try this with `USE=-opengl` or just assume that it only needed it if enabled? Last I checked, recall it uses some GL headers anyway with USE=-opengl, so that's why libglvnd is currently not behind `opengl? (  )` , albeit it may be different for glx.h

On a side-note, I'd have preferred if filed a bug with build.log so it can be analyzed rather than jump to conclusions. There are good odds that it does not actually *need* GL/glx.h and the bit just need to be patched out. I see nothing at all that use libGLX at a glance, even with dlopen, albeit I think it's generating some unused bindings using the header.

For now we could just add the [X] anyway (no need to worry about adding `opengl?` I guess), odds are will review the situation in full with `>=qtwebengine-6.8.9999` when I have time given it may be possible to make all X libraries actually optional in 6.8 and add a USE=X (or at least there was steps toward that,(hard to say if it can work out yet).

Reply:
Well, if *did* try USE=-opengl and it built, guess can do `opengl? ( media-libs/libglvnd[X] )` in DEPEND after all, may avoid unnecessary conflicts for some users.

I'd leave the normal libglvnd that's already there alone though.

Reply:
I don't currently have the patience to rebuild qtwebengine without the `opengl` USE flag, but in a week or so I should have access to a Xeon E5-2690v4 with a tonne of ram. It should build chromium based engines a lot faster than my laptop (8hrs on laptop). I can test any configurations then.

Sadly I don't have a copy of the build log because it got overwritten when I successfully built qtwebengine with media-libs/libglvnd[X]. I should really take snapshots of /var when emerge fails. :disappointed:

I do know that the problem in the build log was `GL/glx.h` was not found, I then did research and realised it is provided by media-libs/libglvnd[X]. I thought requiring `virtual/opengl` would be the elegant way, but I now realise that might not work sometime down the road when things are not X oriented.

For now I'll push `opengl? ( media-libs/libglvnd[X] )`. If you want me to test without opengl, I'll rebuild and test in a few weeks. :slightly_smiling_face: